core/util_test.go

```diff
@@ -224,7 +215,7 @@ func NewDrandTestScenario(t *testing.T, n, thr int, period time.Duration, beacon
 	dt.beaconID = beaconID
 	dt.thr = thr
 	dt.period = period
-	dt.clock = clock.NewFakeClock()
+	dt.clock = clk
```

Comment:
Same reason as above. Also, since the clock is now exposed, we can sync the values correctly across different parts of the code. E.g. We used to have `TestHTTPClientTestFunc` failing because the test clock was internally set to a different time than the client verification clock. Since they are now both on the same clock, which is the `time.Now`, the verifications pass successfully and the test is not flaky anymore.